src/filters/filtersCommon.ml

```diff
@@ -63,8 +63,8 @@ let run_expression_filters ?(ignore_processed_status=false) ctx detail_times fil
 	match t with
 	| TClassDecl c when is_removable_class c -> ()
 	| TClassDecl c ->
+		let ctx = TyperManager.clone_for_module ctx (TypeloadModule.make_curmod ctx.com ctx.g c.cl_module) in
```

Comment:
This is definitely wrong because `ctx` isn't even used other than to mutate `ctx.m` and `ctx.f.curfield` further below. This means that both the `curclass` and `curfield` values come from a captured typer context, and it also means that either the related filters don't care or our tests miss these cases (which is more likely).